PROJECT_PROGRESSION.md

```diff
@@ -206,10 +206,9 @@ Impact Stage projects are expected to participate actively in CPC proceedings, a
 To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
 
  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
- * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
```

Comment:
nit: It will be awesome to link the documents here.

Reply:
We’d need to have template or example documents which we don’t have right now. Maybe file a separate issue about it?

Reply:
This sounds like it's asking for the "contribution processes" to be "laid out in a GOVERNANCE.md file", but I presume that the intent here is for that to be in the CONTRIBUTING.md, which is then referred-to from GOVERNANCE.md, yes?